src/main/java/org/junitpioneer/jupiter/issue/IssueProcessorFactory.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.issue;
+
+import java.util.ServiceLoader;
+
+import org.junitpioneer.jupiter.IssueProcessor;
+
+// Ensures we get the same instance of StoringIssueProcessor in the integration tests
```

Comment:
No correct Javadoc ;)

Reply:
Does this class need javadoc? It's entirely internal to Pioneer.

Reply:
Well, you use the class in another place, so I think it's useful to add JavaDoc (aside from the fact I always add JavaDoc)

Reply:
Our extensions don't have Javadoc either...? 🤷 I think only the public-facing API of Pioneer has proper javadoc.

Ahem. Sure, I can write a quick javadoc. No problemo.